### DISCUSSION TITLE
Does Vim have a list of "known issues"?

### DISCUSSION THREAD

**Discussion by avidseeker** [vim/vim#16103] [Q&A]
Something like https://curl.se/docs/knownbugs.html

**Comment by Shougo** [comment]
`:help todo`

**Reply by saccarosium** [reply]
That can be really out of date. A more reliable source of truth is the issue tracker here on github but currently we don't have something that clear, you need to hunt down issues yourself. 

Might be a something good to do so to improve contributions
